about.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FanExpo San Francisco 2023</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/69940923dd.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="script.js" defer></script>
+  </head>
+  <body>
+    <header class="header-mobile">
+      <nav>
+        <div class="hamburger-menu">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+      </nav>
+    </header>
+    <header class="mobile-menu">
+      <div class="close-icon"><i class="fa-solid fa-xmark"></i></div>
+      <div href="./index.html" class="menu-links">Home</div>
+      <div href=".main-program" class="menu-links">Program</div>
+      <div href="#join-link" class="menu-links">Join</div>
+      <div href="./about.html/#partners-mobile" class="menu-links">Sponsor</div>
+      <div class="menu-links">News</div>
+    </header>
```

Comment:
- [ ] From about page on the mobile version. When clicking on the hamburger, the mobile menu would appear. It is expected to redirect a user from about page to home page when `home` menu item is clicked. But the behaviour in your case is not. It remains on the about page despite clicking on the home menu item number of times. Kindly look into that and fix. 

[mobile-menu-about.webm](https://user-images.githubusercontent.com/26597432/212057082-614db5da-4644-45e4-bf41-d76399b53a8f.webm)